spec/features/course/material/folder_management_spec.rb

```diff
@@ -126,7 +126,8 @@
 
       scenario 'I can view valid subfolders' do
         valid_folders = subfolders.select do |f|
-          f.start_at < Time.zone.now && (f.end_at.nil? || f.end_at > Time.zone.now)
+          f.start_at < Time.zone.now && (f.end_at.nil? || f.end_at > Time.zone.now) &&
+            (f.owner.nil? || f.owner && f.materials.count > 0 && f.children.count > 0)
```

Comment:
Can the `f.owner` condition be removed since it will only be evaluated when `f.owner.nil?` is false, i.e. `f.owner` is present?